### PR TITLE
X962 wildcard support indication on pages

### PR DESCRIPTION
**Pull request by seenanair**
Implemented a generic Information component
  - Displays info icon
  - On hover displays a model with information
  - Allows a prop to display the information required as a component 
  - Added ExternalIDFieldSearchInfo component
 Updated Search and History page with info icon support for External Id

### CODE REVIEW

File: src/components/icons/InfoIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+/**
+ * Remove SVG icon
```
Comment (khelwood) [inline src/components/icons/InfoIcon.tsx:4]: Why does this comment say "Remove SVG icon"?

**Review by khelwood — COMMENTED** [review]
Functionality looks great. I made a couple of edits to the text. I also had a question about a comment.

**Comment by seenanair** [comment]
> Functionality looks great. I made a couple of edits to the text. I also had a question about a comment.

Cool 👍

File: src/components/icons/InfoIcon.tsx (hunk 1 above)
Comment (seenanair) [reply src/components/icons/InfoIcon.tsx:4]: > Why does this comment say "Remove SVG icon"?

Sorry, that needs to be change. I will update it

**Pull request merged**